python/ray/services.py

```diff
@@ -591,21 +574,25 @@ def start_redis_instance(node_ip_address="127.0.0.1",
         Exception: An exception is raised if Redis could not be started.
     """
     assert os.path.isfile(executable)
-    assert os.path.isfile(module)
+    for module in modules:
+        assert os.path.isfile(module)
     counter = 0
     if port is not None:
         # If a port is specified, then try only once to connect.
         num_retries = 1
     else:
         port = new_port()
+
+    # Flatten to ["--loadmodule", "path/m1", ...].
+    loadmodule = list(
+        itertools.chain.from_iterable([["--loadmodule", m] for m in modules]))
```

Comment:
itertools seems too heavy-handed for just this line, and it seems a bit hard to understand. How about

```
load_module_command = []
for module in modules:
    load_module_command += ["--loadmodule", m]
```